decidim-core/spec/system/authentication_spec.rb

```diff
@@ -339,7 +339,7 @@
     end
 
     context "when nickname is not unique case-insensitively" do
```

Comment:
```suggestion
    context "when nickname is not unique" do
```

as we're changing the example we should change the description